apps/alerts/test/cache/store_test.exs

```diff
@@ -79,6 +79,6 @@ defmodule Alerts.Cache.StoreTest do
     alert4 = %Alerts.Alert{id: "126", effect: :cancellation}
 
     Store.update([alert1, alert2, alert3, alert4], nil)
-    assert Store.alert_ids_for_stop_id("543") == ["123", "124"]
+    assert Store.alert_ids_for_stop_id("543") == ["124", "123"]
```

Comment:
Order doesn't matter, but I am unaware of how to compare lists in elixir ignoring the order of the values.  This swap was required to get the test to pass.  The reduce logic stores the order of alerts in descending order (new alerts get appended to the front of the list).  This matches how the routes are stored as well.  
I had debated about changing it, but read some articles about list appending and speed.  They made it seem like combining 2 lists required elixir to traverse the first list to find the end then append the second list to the end (without traversal of the second list).  Appending new alerts to the start of this list is the fast path because the new list to append should be shorter than accumulator list (in most cases)